volatility3/framework/symbols/linux/extensions/network.py

```diff
@@ -199,7 +199,7 @@ def promisc(self) -> bool:
         """
         return self.flags & self._get_net_device_flag_value("IFF_PROMISC") != 0
 
-    def get_net_namespace_id(self) -> int:
+    def _do_get_net_namespace_id(self) -> int:
```

Comment:
Is it really worth keeping the old one around?  No one calls it directly, so might just as well as the exception check into the main body?

Reply:
I like the flow of letting a `_do_*` function do the work then checking its return value (or exception handling usually in Vol3) in the wrapper. Its a convention the Linux kernel follows and keeps the code nice and clean. Its also nice as we aren't changing public APIs that exist already to do this.

Reply:
Ok, I don't have any great stakes in the game so if you want to do it for linux that's fine but it'll expand the API file quite a bit when looking for methods (you'll have to bounce through this facade every time you want to debug it)...